frigate/ptz/autotrack.py

```diff
@@ -238,6 +238,16 @@ def _autotracker_setup(self, camera_config, camera):
         self.move_queues[camera] = queue.Queue()
         self.move_queue_locks[camera] = threading.Lock()
 
+        # handle onvif constructor failing due to no connection
+        if camera not in self.onvif.cams:
+            logger.warning(
+                f"onvif connection to {camera} failed. Disabling autotracking."
+            )
+            camera_config.onvif.autotracking.enabled = False
+            self.ptz_metrics[camera]["ptz_autotracker_enabled"].value = False
+
```

Comment:
```suggestion
```

Reply:
Left the blank line in to be consistent with the other cases in the rest of the function. I can remove the rest of them if you'd prefer that